CODIGO/Protocol.bas

```diff
@@ -1956,6 +1956,7 @@ End Sub
 Private Sub HandleChangeMap()
     On Error GoTo HandleChangeMap_Err
     UserMap = Reader.ReadInt16()
+    ResourceMap = Reader.ReadInt16()
```

Comment:
What is exactly a `ResourceMap` ? Maybe it's the name but I find it confusing. Why is it called `Resource` ?

Reply:
because its refers to the resource name of the map in Resources/maps/

Reply:
I don't think this looks good `Call SwitchMap(UserMap, UserMap)`

Can we make something like `Sub SwitchMap(ByVal map As Integer, optional ByVal NewResourceMap As Integer = map)` 

In this way we avoid calling with the same argument and it looks easier to understand

Reply:
I don't we can default a value to another parameter, just a fixed value